SWLOR.Game.Server/Feature/AIDefinition/GenericAIDefinition.cs

```diff
@@ -72,6 +171,402 @@ static float CalculateAverageHP(uint creature)
                 return (FeatType.ForceStun1, target);
             }
 
+            // Adhesive Grenade
+            if (CheckIfCanUseFeat(self, target, FeatType.AdhesiveGrenade1))
```

Comment:
I think at this point, you're going to want to organize these from highest to lowest. These share a recast timer so you're guaranteed to always get the lowest powered ability.

Reply:
All the NPC's I've made have one version of it. I can go through and reverse them though if needed if down the line we want NPC's to have the ability to use different tiers. However, given how useless any skill other than the max tier of that skill is (force body and med kit excluded) I don't see the value in this.

Other than consistency with other perks in aidefinition, of course, which is fair